.github/workflows/test.yml

```diff
@@ -0,0 +1,43 @@
+name: Test
```

Comment:
Can we rename this to something meaningful?

Reply:
Aha maybe it's referring to running the tests. At first I thought it was just like "Testing workflows". I guess it makes sense then 🙂

Reply:
It's the workflow name shown for example at https://github.com/hugovk/mac-setup/actions:

![image](https://user-images.githubusercontent.com/1324225/102066607-b2780980-3e02-11eb-8194-33773b4a1f32.png)

So some projects have things like "Lint" and so on there. We can change it to whatever we like.